jherey/WeConnect
# Ft user edit and delete review 158786276

#### What does this PR do?
Allows a user edit and delete a review

#### Description of Task to be completed?
- Add controller logic for edit feature
- Add controller logic for delete feature
- Implement edit and delete in the front-end
- Write tests for both endpoints

#### How should this be manually tested?
- Open postman
- Enter endpoint to edit a review: PUT /api/v1/businesses/:businessId/reviews/reviewId
- Login and edit a review
- Success message should be returned
OR
- Enter endpoint to delete a review: DELETE /api/v1/businesses/:businessId/reviews/reviewId
- Login and delete a review
- Success message should be returned

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158786276

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-07-04 at 6 50 59 am" src="https://user-images.githubusercontent.com/32069279/42259197-5651c40a-7f57-11e8-84bd-d1d66fed7a59.png">

<img width="1440" alt="screen shot 2018-07-04 at 6 53 05 am" src="https://user-images.githubusercontent.com/32069279/42259205-5d1defb6-7f57-11e8-9459-703e94059416.png">

<img width="1440" alt="screen shot 2018-07-04 at 6 53 20 am" src="https://user-images.githubusercontent.com/32069279/42259216-641751ea-7f57-11e8-86ac-3f923bae7be7.png">

<img width="1440" alt="screen shot 2018-07-04 at 6 54 24 am" src="https://user-images.githubusercontent.com/32069279/42259219-6a37d66c-7f57-11e8-8a25-b33da7b3a07a.png">

<img width="1440" alt="screen shot 2018-07-04 at 6 54 31 am" src="https://user-images.githubusercontent.com/32069279/42259226-6fe44be0-7f57-11e8-9fec-bc30a00ea853.png">

<img width="1440" alt="screen shot 2018-07-04 at 6 54 36 am" src="https://user-images.githubusercontent.com/32069279/42259234-76c1a48a-7f57-11e8-9754-1f4c4035fd7e.png">
